ykllvmwrap/src/ykllvmwrap.cc

```diff
@@ -40,24 +50,52 @@ struct BitcodeSection {
   size_t len;
 };
 
-// An annotator for `Module::print()` which adds debug location lines.
+// If possible, return a string describing the location of an instruction in
+// the AOT-compiled interpreter source code.
+//
+// Since it is undesirable to show duplicated debug locations, the callee
+// should pass in the last annotation that it saw (in `LastAnnot`), and this
+// function will return `None` if the debug location string was the same as
```

Comment:
Is `None` the right C++ name for this concept?

Reply:
I believe so: https://llvm.org/doxygen/classllvm_1_1Optional.html